docker/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.7'
+services:
+  workout-lol:
+    build: 
+      context: ..
+      dockerfile: docker/dockerfile
+    ports:
+      - 3000:3000
+    depends_on:
+      mongodb:
+        condition: service_healthy
+
+  mongodb:
+    image: mongo:7.0.0-rc6
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=mongoadmin
+      - MONGO_INITDB_ROOT_PASSWORD=secret
+      - MONGO_INITDB_DATABASE=db
+    volumes:
+      - docker/mongo-data:/data/db
```

Comment:
I needed to replace this line with `- ./docker/mongo-data:/data/db` to make it run on my machine

Reply:
You are right, I added it after my tests and forgot to change it back.

Reply:
does it work for you if you follow the steps of the readme? For me it doesn't boot up the workout-lol container. But it could also be a local issue

Reply:
I tested on windows and linux and had no issues (unfortunately I cannot test on mac) could post the error logs?
I also swapped from bindings volumes to docker volumes as the mongo image is running with a UID of 999 which can cause permissions issues (as most of user are 1000).

Reply:
Okay, then I'll approve for merging and will figure out why it's not working on my machine later :)